### PR TITLE
fix(docs): Fixed various letter casing and script errors

### PR DESCRIPTION
**Pull request by deejay1**
## Hey, I just made a Pull Request!

Various fixes around documentation with regard to proprietary names (eg `GitHub` and `macOS`) and missing script parentheses.

#### :heavy_check_mark: Checklist

- [ ] ~A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))~
- [X] Added or updated documentation
- [ ] ~Tests for new functionality and regression tests for bug fixes~
- [ ] ~Screenshots attached (for UI changes)~
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by adowsky — APPROVED** [review]

**Review by camilaibs — APPROVED** [review]
Thanks for contributing 🙌🏻

File: docs/features/software-catalog/configuration.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ catalog (`UrlReaderProcessor`), so no processor configuration is needed. This
 processor _does however_ need an [integration](../../integrations/index.md) to
 understand how to retrieve a given URL. For the example above, you would need to
 configure the [GitHub integration](../../integrations/github/locations.md) to
-read files from github.com.
+read files from GitHub.com.
```
Comment (camilaibs) [inline docs/features/software-catalog/configuration.md:34]:
```suggestion
read files from [github.com](https://github.com/).
```

**Comment by camilaibs** [comment]
Could you please rebase the branch with master, @deejay1?

**Comment by deejay1** [comment]
@camilaibs did it as soon as I got to work :)

**Pull request merged**